sample-metadata/README.adoc

```diff
@@ -148,8 +156,18 @@ In the following example, only if the technical replicate column is provided, on
 | Sample 1          |    run 4   | label free sample | 2                            | 2                            | 000261_C05_P0001563_A00_B00K_F2_TR2.RAW
 |===
 
+NOTE: The comment[technical replicate] column is OPTIONAL and can be easily guess from the data structure if the column `comment[technical replicate]` is not provided. If the same sample has the fraction identifier repeated it can be interpreted as technical replicate. However, some submitters can make more clear that annotation.
```

Comment:
I don't think this is 100% correct. It can only be guessed if the annotator followed some implicit rules like keeping the order of rows.
Note that it can make a difference in practice if we mix up to which replication a file belongs. Let's say you run 2 technical replicates but for whatever reason, the temperature in the instrument room was different in the second run. If you freely mix those files and assign replication run randomly you might end up with completely different results after e.g., normalization.

Reply:
Agree, 'guess' is not the correct term here and we cannot 'guess' what the data provider would consider 'technical replicate' because this is heavily subjective. 
The example of Timo is very useful: files are matched on a certain number of observed variables that should be taken into consideration for interpretation, keeping in mind that unobserved variables like temperature can always confound the analysis. Adding considerations of "replication" is only misleading and should be avoided imho.

Reply:
I have change @timosachsenberg and @mvaudel the phrase that the column can be guessed from the context because it was actually wrong. The current text only defines the way to represent replicates (biological and technical), also it clearly states that is optional.